src/main/resources/messages/messages_zhhk.yml

```diff
@@ -44,68 +43,68 @@ antibot_auto_disabled: '&8[&6用戶系統&8] &3不正常連接數已減少，防
-# TODO recovery_tries_exceeded: 'You have exceeded the maximum number attempts to enter the recovery code. Use "/email recovery [email]" to generate a new one.'
-# TODO recovery_code_correct: 'Recovery code entered correctly!'
-# TODO recovery_change_password: 'Please use the command /email setpassword <new password> to change your password immediately.'
+recovery_code_sent: '&8[&6用戶系統&8] &b帳戶驗證碼已發送到你的郵箱，請查收。'
+recovery_code_incorrect: '&8[&6用戶系統&8] &c帳戶驗證碼無效。 &7(你尚餘 %count 次嘗試機會)'
+recovery_tries_exceeded: '&8[&6用戶系統&8] &c此驗證碼已因多次嘗試後失效，請使用《 /email recovery  <電郵地址> 》重新辦理。'
+recovery_code_correct: '&8[&6用戶系統&8] &a帳戶驗證碼輸入正確。'
+recovery_change_password: '&8[&6用戶系統&8] 請立即使用《 /email setpassword <新密碼> 》指令，以重設你的帳戶密碼。'
 vb_nonActiv: '&8[&6用戶系統&8] &f你的帳戶還沒有經過電郵驗證 ！'
-usage_unreg: '&8[&6用戶系統&8] &f用法： 《 /unregister <密碼> 》'
+usage_unreg: '&8[&6用戶系統&8] &f用法：《 /unregister <密碼> 》'
 pwd_changed: '&8[&6用戶系統&8] &c你成功更換了你的密碼 ！'
 logged_in: '&8[&6用戶系統&8] &c你已經登入過了。'
 logout: '&8[&6用戶系統&8] &b你成功登出了。'
 reload: '&8[&6用戶系統&8] &b登入系統設定及資料庫重新載入完畢。'
-usage_changepassword: '&8[&6用戶系統&8] &f用法： 《 /changepassword <舊密碼> <新密碼> 》'
+usage_changepassword: '&8[&6用戶系統&8] &f用法：《 /changepassword <舊密碼> <新密碼> 》'
 
 # Session messages
 invalid_session: '&8[&6用戶系統&8] &f登入階段資料已損壞，請等待登入階段結束。'
 valid_session: '&8[&6用戶系統&8] &b嗨 ！ 歡迎回來喔～'
 
 # Error messages when joining
 name_len: '&8[&6用戶系統&8] &c你的用戶名不符合規定長度。'
-regex: '&8[&6用戶系統&8] &c用戶名稱錯誤！ 登入系統只接受以下字符： REG_EX'
+regex: '&8[&6用戶系統&8] &c用戶名稱錯誤！ 登入系統只接受以下字符：REG_EX'
 country_banned: '&8[&6用戶系統&8] &4本伺服器已停止對你的國家提供遊戲服務。'
 not_owner_error: '&8[&6用戶系統&8] &4警告！&c你並不是此帳戶持有人，請立即登出。'
 kick_fullserver: '&c抱歉！ 因為伺服器滿人了，所以你目前未能登入伺服器。'
 same_nick: '&8[&6用戶系統&8] &f同名玩家已在遊玩。'
 invalid_name_case: '&8[&6用戶系統&8] &4警告！&c你應該使用「%valid」而並非「%invalid」登入遊戲。'
-# TODO same_ip_online: 'A player with the same IP is already in game!'
+same_ip_online: '&8[&6用戶系統&8] 你正使用的 IP 位址已被其他玩家佔用。'
 
 # Email
-usage_email_add: '&8[&6用戶系統&8] &f用法： 《 /email add <電郵> <重覆電郵> 》'
-usage_email_change: '&8[&6用戶系統&8] &f用法： 《 /email change <舊電郵> <新電郵> 》'
-usage_email_recovery: '&8[&6用戶系統&8] &f用法： 《 /email recovery <電郵> 》'
+usage_email_add: '&8[&6用戶系統&8] &f用法：《 /email add <電郵> <重覆電郵> 》'
+usage_email_change: '&8[&6用戶系統&8] &f用法：《 /email change <舊電郵> <新電郵> 》'
+usage_email_recovery: '&8[&6用戶系統&8] &f用法：《 /email recovery <電郵> 》'
 new_email_invalid: '&8[&6用戶系統&8] &c你所填寫的新電郵地址並不正確。'
 old_email_invalid: '&8[&6用戶系統&8] &c你所填寫的舊電郵地址並不正確。'
 email_invalid: '&8[&6用戶系統&8] &c你所填寫的電郵地址並不正確。'
 email_added: '&8[&6用戶系統&8] &a已新增你的電郵地址。'
 email_confirm: '&8[&6用戶系統&8] &5請重覆輸入你的電郵地址。'
 email_changed: '&8[&6用戶系統&8] &a你的電郵地址已更改。'
 email_send: '&8[&6用戶系統&8] &a忘記密碼信件已寄出，請查收。'
-# TODO email_show: '&2Your current email address is: &f%email'
-# TODO incomplete_email_settings: 'Error: not all required settings are set for sending emails. Please contact an admin.'
+email_show: '&8[&6用戶系統&8] &2你所使用的電郵地址為：&f%email'
+incomplete_email_settings: '&8[&6用戶系統&8] &c電郵系統錯誤，請聯絡伺服器管理員。 &7(err: mailconfig_incomplete)'
```

Comment:
Is there a reasoning behind this? `&7(err: mailconfig_incomplete)`

Reply:
It is intended as the translated string means "Mail server error, please contact server admins." which gives a friendly bummer to player, not to admins. Note that the debug message you quoted is greyed out and is left there for moderators or admins who cannot access the server console instantly, to promptly respond on the incident.

Reply:
Fine by me, thanks for the explanation :)